src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="http://localhost:300api"
+      method="get"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label
+        class="form-field"
+        >
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          Full date of birth:
+          <input
+            type="date"
+            name="date">
+        </label>
+        <label
+          class="form-field"
+        >
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="tearm"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label
+        class="form-field"
+        >
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            class="field"
+          >
+        </label>
+        <label
+        class="form-field"
+        >
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            class="field"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="yes"
+            > Yes
```

Comment:
Move the word to a next line, it'll improve readability. Also, fix it for your next radio
```suggestion
            > 
            Yes
```